picodom.d.ts

```diff
@@ -0,0 +1,30 @@
+declare namespace picodom {
```

Comment:
Thoughts on we following the [module.d.ts UMD example](http://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html) from the [TypeScript handbook](http://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html#umd)?

Basically would just be removing the two declares and adding `export as namespace picodom`;